client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -94,17 +115,23 @@ class EditorMediaModalDetailItem extends Component {
 
 		const mimePrefix = MediaUtils.getMimePrefix( item );
 
-		if ( 'image' !== mimePrefix ) {
+		if ( ! includes( [ 'image', 'video' ], mimePrefix ) ) {
 			return null;
 		}
 
+		let editText = translate( 'Edit Image' );
+
+		if ( 'video' === mimePrefix ) {
+			editText = translate( 'Edit Thumbnail' );
+		}
```

Comment:
Yup, fixed that up in https://github.com/Automattic/wp-calypso/pull/11653/commits/3da40c76eec006510cbfb058db2211018b303e0a.